### ISSUE TITLE
[e2e] Add test which validates only 1 request is sent to the Security Provider API

### ISSUE THREAD

**Issue by seaona** [MetaMask/metamask-extension#18615]
**Context**: there has been an issue, where we sent additional requests to the Security Provider https://github.com/MetaMask/metamask-extension/issues/18560 . This issue has now been fixed by this PR https://github.com/MetaMask/metamask-extension/pull/18598

**What**: add a test which validates that only 1 request is sent to the Security Provider API.

**Comment by bschorchit** [comment]
We'll be deprecating OpenSea's security provider once Blockaid's integration is ready. Should we de-prioritize this? cc: @seaona

**Comment by seaona** [comment]
yes, there is already a PR for the test, but the test won't pass because there was a pending task to solve on Security Provider code -  since I think this won't be fixed as new prio is Blockaid, I guess this ticket and PR could be closed completely @bschorchit
